include/Lv2UridMap.h

```diff
@@ -29,7 +29,7 @@
 
 #ifdef LMMS_HAVE_LV2
 
-#include <lv2/lv2plug.in/ns/ext/urid/urid.h>
+#include <lv2/urid/urid.h>
```

Comment:
Looks like our automated Linux builds can't find this. This is probably another reason why our Linux builds need to be upgraded to ubuntu-20.04 or ubuntu-22.04, since I believe we're still using 18.04.